ripple/src/main/webapp/managebusiness.html

```diff
@@ -0,0 +1,186 @@
+            <input id="business-file" type="file" name="business-file" style="display: none;">
+        </form>
+       <div id="imageCarousel" class="carousel slide" data-interval="false">
+            <div class="carousel-inner row w-100 mx-auto" id="firstRow">
+                <div class="carousel-item col-md-4 active">
+                    <div class="card">
+                    <!-- TODO: Sarah will add the svg image -->
+                      <img class="card-img-top img-fluid" id="card-dynamic-image" src="images/addimage.png" onclick=selectFile('business-file')></img>
+                    </div>
+                </div>
+                <div class="carousel-item col-md-4">
+                    <div class="card">
+                      <img class="card-img-top img-fluid" id="card-dynamic-image" src="images/chinese_food.jpg"></img>
+                    </div>
+                </div>
+                <div class="carousel-item col-md-4">
+                    <div class="card">
+                      <img class="card-img-top img-fluid" id="card-dynamic-image" src="images/louisiana_kitchen.jpg"></img>
+                    </div>
+                </div>
+                <div class="carousel-item col-md-4">
+                    <div class="card">
+                      <img class="card-img-top img-fluid" id="card-dynamic-image" src="images/maison_mathis.jpg"></img>
+                    </div>
+                </div>
+            </div>
+            <a class="carousel-control-prev" href="#imageCarousel" role="button" data-slide="prev">
+            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span>
+            </a>
+            <a class="carousel-control-next" href="#imageCarousel" role="button" data-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+            </a>
+        </div>
+        <div class="col-8">
+            <h3 class="acct-info-subtitle acct-padding-top">Business details</h3>
+            <textarea class="manage-business-text" id="businessName" placeholder="Enter business name..."></textarea>
+            <textarea class="manage-business-text" id="phone" placeholder="Enter business phone..."></textarea>
+            <input class="set-address" id="set-address" type="text" placeholder="Enter business address...">
+        </div>
+        <h3 class="col-8 acct-info-subtitle acct-padding-top">Business tags</h3>
+        <h3 class="col-8 acct-info-subtitle"><small class="text-muted">Select relevant tags from each section to describe your business.</small></h3>
+        <h3 class="col-8 mb-subtitle"><small class="text-muted">Cuisines</small></h3>
+        <div class="d-flex flex-row flex-wrap justify-content-left">
```

Comment:
Just curious if a limit has already been placed on # of tags a user can choose?

Reply:
Not currently I can add it pretty easily though